thorlcr/activities/rollup/throllup.cpp

```diff
@@ -43,7 +43,7 @@ CActivityBase *createDedupRollupActivityMaster(CMasterGraphElement *container)
     {
         if (TAKdedup == container->getKind())
         {
-            if (((IHThorDedupArg *)container->queryHelper())->compareAll())
+            if (((IHThorDedupArg *)container->queryHelper())->getFlags() & DFcompareall)
```

Comment:
I've made the change already.. probably, not worth going back and doing this now.